diff --git a/Makefile b/Makefile
index 8bd91de..d81a1ed 100644
--- a/Makefile
+++ b/Makefile
@@ -145,6 +145,7 @@ venv: venv/created
 venv/created: ./requirements.txt ./requirements_dev.txt ./requirements_lint.txt ./dependencies/python/precompile_time.txt ./dependencies/python/run_time.txt 
 	rm -rf ./venv
 	$(PYTHON_BIN) -m venv ./venv
+	PYTHONPATH= PYTHON_BIN=$(PYTHON_BIN) ./hotfix_pyyaml.sh
 	PYTHONPATH= ./venv/bin/$(PYTHON_BIN) -m pip install --upgrade --no-cache-dir pip
 	PYTHONPATH= ./venv/bin/$(PYTHON_BIN) -m pip install --upgrade --no-cache-dir -r ./requirements_dev.txt
 	touch $@
